modules/oadp-usecase-include-ca-cert-backup.adoc

```diff
@@ -0,0 +1,274 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-use-cases/oadp-usecase-enable-ca-cert.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-usecase-backup-restore-ca-cert_{context}"]
+= Backing up an application by including a self-signed CA certificate
```

Comment:
```suggestion
= Backing up an application and its self-signed CA certificate
```
The "including" dangles a bit, but adding more text would mean a longer title, which is not ideal.